ivy_tests/test_ivy/test_misc/test_assertions.py

```diff
@@ -40,16 +40,14 @@
 def test_check_all(results):
     filename = "except_out.txt"
     orig_stdout = sys.stdout
-    f = open(filename, "w")
-    sys.stdout = f
-    lines = ""
-    try:
-        check_all(results)
-    except Exception as e:
-        print(e)
-    sys.stdout = orig_stdout
-    f.close()
-
+    with open(filename, "w") as f:
```

Comment:
let's add a new line before and after this block similar to how it's done with the next blocks in this function. Same applies to the other changes made

Reply:
@vedpatwardhan sir, I made the suggested changes, Please look into it now, Thank you.